guide/english/linux/index.md

```diff
@@ -10,27 +10,27 @@ Linux is a name which broadly denotes a family of free and open-source software
 
 The development of Linux is one of the most prominent examples of free and open-source software collaboration. The underlying source
-code may be used, modified and distributed commercially or non-commercially by anyone under the terms of its respective licenses, such
+the code may be used, modified and distributed commercially or non-commercially by anyone under the terms of its respective licenses, such
 as the GNU General Public License.
 
 Linux was first started as a clone of the MINIX operating system. Open source contributors added onto the kernel and expanded its
 hardware compatibility. Linux's hardware support and free licensing made Linux a popular choice for desktop and server computing in the
-mid 90s. Today, Linux is the most popular operating system in the world. It powers 90% of the world's servers, and is the basis of
+mid-90s. Today, Linux is the most popular operating system in the world. It powers 90% of the world's servers and is the basis of
 Google's popular Android operating system.
 
 There are also many versions of Linux designed to be run on personal computers, such as Debian or Ubuntu. More importantly, these
 different versions of Linux (called distributions or distros for short) allow the user varying degrees of personalization and control of the operating
 system. This means that users can choose their Linux distributions based on their wants and needs.
 
-There is also Kali Linux which is used for advanced penetration testing and auditing. It has over 700 tools, to be able to use itefficiently you will need to know how to use command line. Since, it used for ethical hacking and pen testing you will have to perform certain malicious hacks which might brick your system. Therefore, one should have complete knowledge about the command line interface, so, that you can fix errors if occured and knew what you have done and hence it is not advised for beginners.
+There is also Kali Linux which is used for advanced penetration testing and auditing. It has over 700 tools, to be able to use it efficiently you will need to know how to use the command line. Since it used for ethical hacking and pen testing you will have to perform certain malicious hacks which might brick your system. Therefore, one should have complete knowledge about the command line interface, so, that you can fix errors if occurred and knew what you have done and hence it is not advised for beginners.
```

Comment:
```suggestion
There is also Kali Linux which is used for advanced penetration testing and auditing. It has over 700 tools, and to be able to use it efficiently, you will need to know how to use the command line. Since it used for ethical hacking and pen testing, you will have to perform certain malicious hacks which might brick your system. Therefore, one should have complete knowledge about the command line interface, so that you can fix errors if they occur and know what you have done, hence it is not advised for beginners.
```